### ISSUE TITLE
Keras benchmark should not modify ~/.keras/keras.json

### ISSUE THREAD

**Issue by mratsim** [aizvorski/vgg-benchmarks#9]
It crashed and burned on argparsing on my computer and left me with an empty configuration file.

There is potential of data loss, it should at least prompt, or even better, just display a message about the expected configuration.

**Comment by alex-at-thimble** [comment]
@mratsim You are right, but it isn't obvious how else to set keras backend on the fly.  Do you have a suggestion?

The original keras.json is backed up, and restored after the benchmark runs; it mentions that on the front page.  I know this is not ideal, but I don't know how else to accomplish the same effect.

**Comment by mratsim** [comment]
You can use the KERAS_BACKEND environment variable which takes precedence over the config file according to the [documentation](https://keras.io/backend/):

KERAS_BACKEND=tensorflow python -c "from keras import backend"

**Comment by aizvorski** [comment]
Fixed in 4f551e19c89a83df0f9e0e6156d53e6e37463a8e
